tests/workflows/basic_await_any/workflow.py

```diff
@@ -25,6 +26,9 @@ def run(self) -> Outputs:
 class AwaitAnyNode(BaseNode[BaseState]):
     top = TopNode.Outputs.total
 
+    class Trigger(BaseNode.Trigger):
+        merge_behavior = MergeBehavior.AWAIT_ANY
```

Comment:
This test was always busted. It claimed to be an AwaitAny node, but was still using the default AWAIT_ATTRIBUTES strategy